tempesta_fw/sock_srv.c

```diff
@@ -521,17 +528,15 @@ tfw_srv_conn_free(TfwSrvConn *srv_conn)
 }
 
 static int
-tfw_sock_srv_add_conns(TfwServer *srv, int conns_n)
+tfw_sock_srv_add_conns(TfwServer *srv)
 {
 	int i;
 	TfwSrvConn *srv_conn;
 
-	for (i = 0; i < conns_n; ++i) {
+	for (i = 0; i < srv->conn_n; ++i) {
 		if (!(srv_conn = tfw_srv_conn_alloc()))
 			return -ENOMEM;
-		tfw_connection_link_peer((TfwConn *)srv_conn,
-					 (TfwPeer *)srv);
-		tfw_sg_add_conn(srv->sg, srv, srv_conn);
```

Comment:
I meant something like if we reconfigure just number of connections for existing server groups, then probably we'd need the specific call to add a connection. However, I'm not sure whether we should introduce separate workflow for the case. Maybe it's easier just to reconfigure the server groups without significant overhead. If you think that we don't need the call any more, then I'm fine with it.